packages/core/admin/admin/src/hooks/useSettingsMenu/index.js

```diff
@@ -89,20 +91,21 @@ const useSettingsMenu = () => {
     const sections = formatLinks([
       {
         ...settings.global,
-        links: sortLinks([...settings.global.links, ...globalLinks]).map((link) => ({
+        links: sortLinks([...settings.global.links, ...globalLinks.map(addPermissions)]).map((link) => ({
           ...link,
           hasNotification: link.id === '000-application-infos' && shouldUpdateStrapi,
         })),
       },
       {
         id: 'permissions',
         intlLabel: { id: 'Settings.permissions', defaultMessage: 'Administration Panel' },
-        links: adminLinks,
+        links: adminLinks.map(addPermissions),
       },
       ...Object.values(otherSections),
     ]);
 
     getData();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This is probably about `addPermissions`. Could you please add it to the dependency array instead?